Modules/SubModule_Population/xASL_stat_GetDICOMStatistics.m

```diff
@@ -154,11 +154,27 @@ function xASL_stat_GetDICOMStatistics(x, ScanType, HasSessions, bOverwrite)
 
 %% 3) Write TSV file
 
-% Remove empty elements
-TSV(cellfun('isempty',TSV)) = {'_'};
-
-% Remove nan elements
-TSV(cell2mat(cellfun(@(x)any(isnan(x)),TSV,'UniformOutput',false))) = {'n/a'};
+% Ensure all elements of the TSV cell array have the correct format
+for iRow=1:size(TSV,1)
+    for iColumn=1:size(TSV,2)
+        % Remove lists (we had a problem where it was tried to insert SliceTiming arrays into the table)
+        if size(TSV{iRow,iColumn},1)>1
+            TSV{iRow,iColumn} = 'n/a';
+        end
+        % Remove empty elements (Empty cells in the TSV can lead to reading errors)
+        if isempty(TSV{iRow,iColumn})
+            TSV{iRow,iColumn} = '_';
```

Comment:
Because we write this table where we have empty places and if we write '' to the cell array some tsv readers crash. We did this before and it looks quite okay I think.

Reply:
I don't like it, but for that use case it's the cleanest solution IMHO

Reply:
I like this strategy, I just think that '_' is arbitrary, whereas BIDS defaults to 'n/a' for missing values. So it seems we should use this as well right?

Reply:
I introduced the '_' only for these placeholder cells that you used in these combined tables. All the numerical values should be defaulted to 'n/a'. Check out the example below:

![image](https://user-images.githubusercontent.com/46593074/129586853-20e8b172-23c6-42cc-b43a-6533cef4e6cc.png)

Reply:
I am fine with that. I would merge as is and change in the future if we find this weird. Unless Henk has a different opinion.

Reply:
Ah I see, nice. So all missing values, irrespective their meaning/format, will be 'n/a', according to BIDS. Only the empty placeholder cells will be an underscore '_', that looks good.